ext/standard/user_filters.c

```diff
@@ -366,16 +368,16 @@ PHP_FUNCTION(stream_bucket_make_writeable)
 		RETURN_THROWS();
 	}
 
-	ZVAL_NULL(return_value);
-
 	if (brigade->head && (bucket = php_stream_bucket_make_writeable(brigade->head))) {
 		ZVAL_RES(&zbucket, zend_register_resource(bucket, le_bucket));
-		object_init(return_value);
+		object_init_ex(return_value, stream_bucket_class_entry);
 		add_property_zval(return_value, "bucket", &zbucket);
 		/* add_property_zval increments the refcount which is unwanted here */
 		zval_ptr_dtor(&zbucket);
 		add_property_stringl(return_value, "data", bucket->buf, bucket->buflen);
 		add_property_long(return_value, "datalen", bucket->buflen);
```

Comment:
Can't you set the property directly? I have something that I remember that I wanted to get rid of those because the ``property_add`` APIs did something weird... maybe editing the HashTable directly but not exactly sure.

Reply:
Do you mean to use `zend_update_property_*()`? As far as I see the only difference between this and `add_property_*` is that the former supports setting a fake scope. But otherwise both of them use the property handler to write the property.

That said, I'm fine with using `zend_update_property_*()` API.

Reply:
Ah no the thing was about using known zend_strings https://github.com/php/php-src/pull/11387 I think either is fine and I need to go back to that PR.